### DISCUSSION TITLE
How to print current time in bt script

### DISCUSSION THREAD

**Discussion by Engg-blogger** [bpftrace/bpftrace#3399] [General]
Hi Team - I am trying to capture the current date and time in one of the bpftrace script. However, most of the example are using strftime which gives elapsed time in nanoseconds. Could you provide an example?

**Comment by ZhangYet** [comment]
Can this example meet your demand?

```shell
sudo bpftrace -e 'BEGIN { print(strftime("%D %H:%m:%S", nsecs)) }'

Attaching 1 probe...
08/18/24 01:08:31
```

**Reply by Yenya** [reply]
Hi all, just a minor correction: for minutes, you need an uppercase "%M" instead of lowercase "%m", which is month number (08 in the above example).

**Reply by jordalgo** [reply]
Ah yes, thanks @Yenya
